src/lua/uri.c

```diff
@@ -301,7 +301,7 @@ luaT_uri_create_internal(lua_State *L)
 {
 	struct uri *uri = (struct uri *)lua_topointer(L, 1);
 	if (uri == NULL)
-		luaL_error(L, "Usage: uri.internal.uri_create(string|table)");
+		luaL_error(L, "Usage: uri_lib.uri_create(string|table)");
```

Comment:
Looks like a typo: `s/uri_lib/uri.lib/`.

Reply:
Not exactly. I assume something like `local uri_lib = require('uri.lib')`. If I'll write `uri.lib` it would mean that there is a table `uri`, which has the `lib` field. But there is no such table.

`uri_lib` is not really nice, because there is nothing named `uri_lib` in the public API. I hope that this error shouldn't occur at any more-or-less valid usage. However, if you have some idea how to make the error message better, I'm open for it.

Reply:
I looked over `src/` directory and found a big variety of names, depending on the function type: public/private, function/method, etc. All in all, I'm OK with your choice, so feel free to ignore.

Reply:
Let me know if you'll have a proposal that looks definitely better than what I implemented now (I see no one currently). Closing the discussion for now.